pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -586,6 +586,14 @@ var _ = Describe("Pod Network", func() {
 			})
 		})
 		Context("Macvtap plug", func() {
+			BeforeEach(func() {
+				primaryPodInterface.HardwareAddr = fakeMac
+			})
+
+			AfterEach(func() {
+				primaryPodInterface.HardwareAddr = nil
```

Comment:
It is not required, `primaryPodInterface` is override on the main `beforeEach`.

Reply:
While a bit redundant, I think it is nice to highlight that the operation is symmetric (I'm changing something on a test setup; I'm putting it back in the test teardown).

If you insist, I can get rid of this and make it asymmetric (I just think it is not a good practice to have asymmetric test setup / teardown; having it explicit simplifies my life).

Reply:
I am totally lost, probably because the tests here are hiding the usage of this one.
Can you please point me to where this takes affect?

Reply:
I'm merely stating that if I update the primary pod interface to have a mac address I'll need to update the expecters to expect the creation of the primary pod interface with a mac address. Something which I'd like to avoid, mostly because it adds nothing but noise.

The easiest way out of this is probably to just use a different interface for the macvtap tests, and stop touching that generic one.

Reply:
I've had no reply so just went ahead and did it.